README.md

```diff
@@ -21,30 +21,30 @@ To add the toolbox to a project, you add the package to the csproj project file:
 
```

Comment:
Best deze file verwijderen en enkel de lower cased readme overhouden

Reply:
Raar... geen idee hoe dat gebeurd is. upper cased readme is verwijderd